osm2gtfs.py

```diff
@@ -31,18 +27,32 @@
 
 
 def main():
+
+    # Load config json file
+    try:
+        config = json.load(args.config)
```

Comment:
The error handling isn't really working here when the script is called without parameters:
```
  File "./osm2gtfs.py", line 33, in main
    config = json.load(args.config)
  File "/usr/lib64/python2.7/json/__init__.py", line 287, in load
    return loads(fp.read(),
AttributeError: 'NoneType' object has no attribute 'read'
```
I think it should take `config.json` by default and show a proper error message when the file can not be found.